### ISSUE TITLE
More nitter instances to support?

### ISSUE THREAD

**Issue by Impeta** [qsniyg/maxurl#1105]
Your script already supports Nitter, though still not all of its instances. Ideally, you could try make it dynamic and automatically have them all as supported at once, given the instances are hardly any different from another! Is this possible?

**Comment by qsniyg** [comment]
Good idea, a generic regex should be possible without breaking other sites (and if it does, we can just add an exception :)
